test/helpers/api-v3-integration.helper.js

```diff
@@ -116,10 +132,16 @@ export async function generateUser (update = {}) {
 // will will become the groups leader. Takes an update
 // argument which will update group
 export function generateGroup (leader, details = {}, update = {}) {
+  details.type = details.type || 'party';
+  details.privacy = details.privacy || 'private';
+  details.name = details.name || 'test group';
```

Comment:
The only thing that comes to my mind with it being a party is that you can only have one for user. But I think it's more than :+1: since creating a guild is really easy